example/config/AnalogMeterConfig.json

```diff
@@ -35,21 +35,6 @@
       },
       "cancelOnResult": true
     },
-    "nativeBarcodeEnabled": true,
-    "segment": {
```

Comment:
The presence of `segment` causes an onscreen toggle to select between different (scan) modes to appear. This is nonsensical because this config (which presents Analog and Digital modes) is explicitly loaded when the user selects the Analog mode. Not only are the two modes deprecated, but even if they aren't, there's a tile doing exactly the other example case (in this case Digital).

Reply:
So it would be better if we remove it.